templates/UmbracoProject/.template.config/ide.host.json

```diff
@@ -37,7 +36,8 @@
     },
     {
       "id": "DevelopmentDatabaseType",
-      "isVisible": true
+      "isVisible": true,
+      "persistenceScope": "templateGroup"
```

Comment:
Adding this `persistenceScope` value will make your IDE remember the last used value, making it easier to set your preferred options once and re-use that for all future projects created from this template.